map-storage/src/MapsManager.ts

```diff
@@ -104,8 +106,29 @@ class MapsManager {
         return gameMap;
     }
 
-    public clearAfterUpload(key: string): void {
+    public async clearAfterUpload(key: string): Promise<void> {
         console.log(`UPLOAD/DELETE DETECTED. CLEAR CACHE FOR: ${key}`);
+        const areas = this.loadedMaps.get(key)?.getGameMapAreas()?.getAreas().values();
+
+        if (areas) {
+            const promises = Array.from(areas).reduce((acc, currArea) => {
+                currArea.properties.forEach((property) => {
+                    const resourceUrl = property.resourceUrl;
+                    if (resourceUrl) {
+                        acc.push(limit(() => _axios.delete(resourceUrl, { data: property })));
+                    }
+                });
+                return acc;
+            }, [] as Promise<unknown>[]);
+
+            try {
+                await Promise.all(promises);
+            } catch (error) {
+                console.error("Failed to execute all request on resourceUrl", error);
+                Sentry.captureMessage(`Failed to execute all request on resourceUrl ${JSON.stringify(error)}`);
+            }
+        }
+
```

Comment:
This probably does not belong here. clearAfterUpload is specifically meant to clear maps that are in memory.

But someone could delete a map when there is noone inside.

You should move this piece of code in here: https://github.com/thecodingmachine/workadventure/blob/a0d14c3d48cc465328116005b9bbf25511d42a56/map-storage/src/Upload/UploadController.ts#L628-L628